CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## [v3.2.0](https://github.com/gordalina/GordalinaMixpanelBundle/releases/tag/3.1.0)
+
+[3.1.0...3.2.0](https://github.com/gordalina/GordalinaMixpanelBundle/compare/3.1.0...3.2.0)
+
+### Features
+
+- Used `Profile properties`. [#16](https://github.com/gordalina/GordalinaMixpanelBundle/pull/16) by @RomulusED69
+
+### Deprecations
+
+- In your config file replace
+
+```yaml
+Symfony\Component\Security\Core\User\UserInterface:
+  id: username
+  email: email
+YourAppBundle\Entity\User:
```

Comment:
```suggestion

App\Entity\User:
```

With the newline and with `App` instead of `YourAppBundle`.
Symfony 4 and 5 (version only supported by this bundle) are bundle-less structure. 
99.9% of the time you will use `App\Entity\User`

Same suggestion for the following Yaml block (and for the README too :D)